BParkingNano/plugins/PATLowPtElectronSeedingEmbedder.cc

```diff
@@ -46,19 +51,25 @@ void PATLowPtElectronSeedingEmbedder::produce(edm::StreamID, edm::Event &evt, ed
   evt.getByToken(ptBiased_src_, ptBiased);
   edm::Handle<edm::ValueMap<float> > unBiased;  
   evt.getByToken(unBiased_src_, unBiased);
+  edm::Handle<edm::ValueMap<float> > mvaId;  
+  evt.getByToken(mvaId_src_, mvaId);
 
 
   std::unique_ptr<pat::ElectronCollection> out(new pat::ElectronCollection);
 
-  for(auto ele : *lowpt) {
-    const reco::GsfTrackRef gsfTrk = ele.gsfTrack();
+  for ( size_t iele = 0; iele < lowpt->size(); ++iele ) {
+    edm::Ref<pat::ElectronCollection> ref(lowpt,iele);
+    const reco::GsfTrackRef gsfTrk = ref->gsfTrack();
     float unbiased_seedBDT = float((*unBiased)[gsfTrk]);
     float ptbiased_seedBDT = float((*ptBiased)[gsfTrk]);
+    float mva_id = float(mvaId->get(ref.key()));
```

Comment:
@bainbrid then maybe we can go for a simpler ```(*mvaId)[ref]```

Reply:
Done. PR is updated.